### PR TITLE
Fix database creation about license foreign key

### PR DESCRIPTION
**Pull request by RemiBauzac**
The foreign key is not built in the right table in the code, and the SQL script is not up to date.

### CODE REVIEW

File: dbmodel/database script.sql [hunk 1]
```diff
@@ -2,7 +2,9 @@
 CREATE TABLE IF NOT EXISTS content (
   id varchar(255) PRIMARY KEY NOT NULL,
   encryption_key varchar(64) NOT NULL,
-  location text NOT NULL
+  location text NOT NULL,
+  length int(11) NOT NULL,
```
Comment (danielweck) [inline dbmodel/database script.sql]: Currently as `bigint`
https://github.com/readium/readium-lcp-server/pull/106/files#diff-71ebe2bcf31a915b1fa3b3b289d5d31dR112
OK?

Comment (snemegeer) [inline dbmodel/database script.sql]: I would keep the bigint , int(11)  will be limited by 32 bit underlying structure

Comment (snemegeer) [inline dbmodel/database script.sql]: aren't tables created in the go projects? I don't remember using this file ...

Comment (danielweck) [inline dbmodel/database script.sql]: `CREATE TABLE IF NOT EXISTS` in `index/index.go` uses `length bigint`, so `int(11)` should be consistent. I am okay with `bigint`. @RemiBauzac ?

**Comment by llemeurfr** [comment]
Finally, I updated index.go and store.go in @d6881e4.

I wonder if the initialization script should be updated or deleted, as the creation of tables is conditional and made is the golang code (not pretty but works).

**Pull request closed without merging**